.copyplop.yaml

```diff
@@ -0,0 +1,49 @@
+# Example configuration for IBM project
+copyright:
+  holder: "IBM Corp."
+  start_year: 2014
+  current_year: 2025
+  format: "Copyright {{.Holder}} {{.StartYear}}, {{.CurrentYear}}"
+
+license:
+  enabled: true
+  identifier: "MPL-2.0"
+  format: "SPDX-License-Identifier: {{.Identifier}}"
+
+files:
+  include_paths:
+    - "internal/service/[a-d]**"
+    - "internal/service/[t-z]**"
```

Comment:
If we're partially enabling this, should `copywrite` also remain in place for the other service paths?